pkgs/by-name/mi/minutor/package.nix

```
@@ -0,0 +1,43 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  qt5,
+  zlib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "minutor";
+  version = "2.21.0";
+
+  src = fetchFromGitHub {
+    owner = "mrkite";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "minutor";
```

Excessive reference to finalAttrs will increase the eval time, the pattern here is longer, and the reference here has no logical correspondence (in contrast with `version`)

---

There's no measurable difference between having both `repo` and `rev` depend on `finalAttrs` and just `rev`. I even had a benchmark run where it was slightly faster *with* `repo` being dynamic.